Documentation/En/Index.xyl

```diff
@@ -97,7 +97,7 @@
   <p>The WebSocket protocol runs on TCP, thus we will start a local WebSocket
   server on the 8889 port:</p>
   <pre><code class="language-php">$server = new Hoa\Websocket\Server(
-    new Hoa\Socket\Server('tcp://127.0.0.1:8889')
+    'tcp://127.0.0.1:8889'
```

Comment:
We must show that we can inject `Hoa\Socket\Server` or use a string that will instantiate this object for us.